lib/sqlite-store.js

```diff
@@ -273,26 +385,45 @@ module.exports = function (options) {
 
     stm.text = 'INSERT INTO ' + escapeStr(table) + ' (' + escapeStr(fields) + ') values (' + escapeStr(params) + ')'
     stm.values = values
+
     return stm
   }
 
 
-  var updatestm = function (ent) {
-    var stm = {}
-    var table = tablename(ent)
-    var fields = ent.fields$()
-    var entp = makeentp(ent)
+  var updatebyidstm = function (ent, schema) {
+    const stm = {}
+    const table = tablename(ent)
+    const entp = makeentp(ent)
 
-    var values = {}
-    var params = []
+    const values = {}
+    const params = []
 
-    fields.forEach(function (field) {
-      if (!(_.isUndefined(ent[field]) || _.isNull(ent[field]))) {
-        var fieldPlaceholder = '$' + field
-        values[fieldPlaceholder] = entp[field]
-        params.push(field + ' = ' + fieldPlaceholder)
+    const should_merge = (function () {
+      if ('merge$' in ent) {
```

Comment:
The `merge$` qualifier passed in the query takes precedence over the Seneca's global `merge` option. This allows users to override the app-global `merge`, if necessary.